setup.cfg

```diff
@@ -4,10 +4,45 @@ cover-package=kivy
 with-id=1
 verbosity=2
 logging-level=DEBUG
+
 [kivy]
 cython_min=0.24
 cython_max=0.29.14
 cython_exclude=0.27,0.27.2
+
 [coverage:run]
 plugins =
     kivy.tools.coverage
+
+[options]
+install_requires =
+    Kivy-Garden>=0.1.4
+    docutils
+    pygments
+dependency_links = https://github.com/kivy-garden/garden/archive/master.zip
+
+[options.extras_require]
+tuio = oscpy
+dev =
+    pytest>=3.6
+    pytest-cov
+    pytest_asyncio
+    pyinstaller
```

Comment:
i was wondering if this was a bit heavy handed, but actually, i guess it's fine for dev mode, people can always install the parts they are interested in if they want to pick and choose.

Reply:
Yes, precisely. I wasn't sure about this either, but in the CI we use it for some pytests so I included it.